registry/example/example_test.go

```diff
@@ -0,0 +1,69 @@
+package example
+
+import (
+	"context"
+	"log"
+	"testing"
+
+	eh "github.com/looplab/eventhorizon"
+	eventstore "github.com/looplab/eventhorizon/eventstore/memory"
+	eventpublisher "github.com/looplab/eventhorizon/publisher/local"
+
+	"github.com/looplab/eventhorizon/registry"
+)
+
+// TestDomain bad integration test
+func TestDomain(t *testing.T) {
+	eventStore := eventstore.NewEventStore()
+	eventPublisher := eventpublisher.NewEventPublisher()
+
+	// eventbus := local.
+	r, err := registry.NewBuilder().
```

Comment:
This reads pretty nicely! An idea could be to somehow start from the Aggregate instead, maybe:
```
Aggregate(NewHereAggregate).
HandlesEvent().
HandlesEvent().
Build()
```

In that case multiple "builds" can be created and attached to a standard command bus. That would maybe require the feature described in #54. If done like this, the command bus can be used (as normally done) to dispatch commands.